Documentation/DomainModel/Entities.md

```diff
@@ -0,0 +1,209 @@
+# Domain Entities
+
+## Identity Context
+
+### Administrator
+Table name: `phplist_admin`
+
+An **administrator** can log in to the system and is allowed to administer
+selected **lists** (as the owner), send **campaigns** to these lists and edit
+**subscribers**.
+
+Administrators are not subscribers. If administrators would like to subscribe
+to lists, they need to have a separate subscriber account.
+
+### AdministratorAttribute
+Table name: `phplist_adminattribute` or `phplist_admin_attribute` 
+
+What does this do? How does this work?
+
+### AdministratorPasswordRequest
+Table name: `phplist_admin_password_request`
+
+What does this do? How does this work?
+
+### AdministratorToken
+Table name: `phplist_admintoken`
+
+This table contains the API tokens for **administrators**. Those API tokens are
+used for access to the REST API.
+
+
+## SubscriptionContext
+
+### Attribute
+Table name: `phplist_user_attribute`
+
+An **attribute** is a field for subscribers. This entity does not
+contain the values for this attribute for each individual subscribe, but
+only the name of the attribute and an ID. 
+
+### AttributeValue
+Table name: `phplist_user_user_attribute`
+
+An **attributeValue** contains the value for a particular **attribute** for a
+particular **subscriber**.
+
+### SubscribePage
+Table name: `phplist_subscribepage`
+
+What does this do? How does this work?
+
+### SubscribePageData
+Table name: `phplist_subscribepage_data`
+
+What does this do? How does this work?
+
+### Subscriber
+Table name: `phplist_user_user`
+
+A subscriber can subscribe to multiple **lists** and can receive email
+messages from **campaigns** for those lists.
+
+Each subscriber has an email address.
+
+To manage their list subscriptions and preferences, subscribers don't
+generally authenticate, except with a token, although you can switch on
+requiring passwords (off by default).
+
+A subscriber can have some **attributeValues**.
+
+### SubscriberBlacklist
+Table name: `phplist_user_blacklist`
+
+What does this do? How does this work?
```

Comment:
Yes, I think suppression-list is now a fairly common term in the field. Let's stick with that.